connectivity/tests/errors.go

```diff
@@ -61,17 +65,32 @@ func (n *noErrorsInLogs) Name() string {
 	return "no-errors-in-logs"
 }
 
+type podID struct{ Cluster, Namespace, Name string }
+type podInfo struct {
+	containers []string
+	client     *k8s.Client
+}
+
 func (n *noErrorsInLogs) Run(ctx context.Context, t *check.Test) {
 	var since time.Time
-	ct := t.Context()
 
-	for _, pod := range ct.CiliumPods() {
-		pod := pod
-		logs, err := pod.K8sClient.CiliumLogs(ctx, pod.Pod.Namespace, pod.Pod.Name, since, nil)
-		if err != nil {
-			t.Fatalf("Error reading Cilium logs: %s", err)
+	pods, err := n.allCiliumPods(ctx, t.Context())
+	if err != nil {
+		t.Fatalf("Error retrieving Cilium pods: %s", err)
+	}
+
+	for pod, info := range pods {
+		client := info.client
+		for _, container := range info.containers {
+			id := fmt.Sprintf("%s/%s/%s (%s)", pod.Cluster, pod.Namespace, pod.Name, container)
```

Comment:
Since we have a podID type perhaps a convenience Stringer on this type would lean this out?

Reply:
I'd personally keep it easy, given that this is used only once.